packages/docs/src/pages/guidelines/accessibility.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Accessibility
+---
+
+CMS strives to make its Design System work for as many people as possible, including people with disabilities.
+
+## What we do to help meet and exceed standards
+
+- build accessibility into our [guiding principles](https://github.com/CMSgov/design-system/blob/master/guides/GUIDING-PRINCIPLES.md).
+- include accessibility as part of our design process when working on new or existing components.
+- provide accessibility as a [checkpoint in new components](https://github.com/CMSgov/design-system/blob/master/.github/ISSUE_TEMPLATE/component-validation-checklist.md) created in the design system.
```

Comment:
Ahh, I'm conflicted on this. Accessibility is included in that issue template, but we never incorporate that into our process. Issue templates are only used by external people on github, and we've never seen anyone file an issue with that template, or with code introducing a new component. The issue template almost feels like false advertising, or a process that is technically true, but in practice meaningless.

Reply:
I think @davidakennedy had some plans for this accessibility checklist unless I'm mistaken?